cmd/collector/app/span_processor_test.go

```diff
@@ -400,27 +400,74 @@ func TestSpanProcessorWithCollectorTags(t *testing.T) {
 }
 
 func TestSpanProcessorCountSpan(t *testing.T) {
-	mb := metricstest.NewFactory(time.Hour)
-	m := mb.Namespace(metrics.NSOptions{})
+	tests := []struct {
+		name                  string
+		enableDynQueueSizeMem bool
+		enableSpanMetrics     bool
+		expectedUpdateGauge   bool
+	}{
+		{
+			name:                  "enable-dyn-queue-size-enable-metrics",
```

Comment:
```suggestion
			name:                  "enable dyn-queue-size, enable metrics",
```
nit: remove dashes for better readability, separate with commas